main.nf

```diff
@@ -81,17 +83,24 @@ if (!seq_date) {
   throw new Exception("${major_release} release not supported in map variables in nf code.")
 }
 
-// If major release is not TEST
-// Specify production synapse id to pass into processing
-if (major_release != "TEST") {
-  project_id = "syn3380222"
-  center_map_synid = "syn10061452"
-  is_prod = true
-}
-else {
+// define whether to run TEST, STAGING or PRODUCTION
+if (major_release == "TEST") {
   project_id = "syn7208886"
   center_map_synid = "syn11601248"
   is_prod = false
+  is_staging = false
+} else if (major_release == "STAGING"){
+  project_id = "syn22033066"
```

Comment:
Eventually - it'd be nice if we can just use the synapse id to determine if it's staging or prod (within the code, not the Nextflow pipeline)

Reply:
https://sagebionetworks.jira.com/browse/GEN-1431